lib/screens/preferences/preferences_screen.dart

```diff
@@ -42,31 +36,43 @@ class _PreferencePageState extends State<PreferencePage> {
     super.dispose();
   }
 
-  Widget buildStepWithNavBar(Widget child) {
+  Map<Enum, String> get _localizedConfigSteps =>
+      localizeEnum(context, ConfigSteps.values, _localization.preferenceTabs);
```

Comment:
Isn´t this more readable? 
```suggestion
      Map<ConfigSteps, String> preferencesSteps = {
  ConfigSteps.general: _localization.general,
  ConfigSteps.wallet: _localization.wallet
};
```